packages/firebase/index.d.ts

```diff
@@ -6060,6 +6061,20 @@ declare namespace firebase.firestore {
      */
     app: firebase.app.App;
 
+    /**
+     * Clears the persistent storage.
+     *
+     * Must be called after the app is shutdown or before other methods (other
```

Comment:
It's OK to mention the idea of "not started", which could give users some conceptual purchase on what we're trying to do here. What I was pointing out earlier was that describing this restriction only as not started without defining what that meant would be hard for someone who doesn't live and breathe this stuff to understand.